tests/Parser/DecimalMoneyParserTest.php

```diff
@@ -115,6 +115,8 @@ public function formattedMoneyExamples(): array
             ['9.999', 'USD', 2, 1000],
             ['9.99', 'USD', 2, 999],
             ['-9.99', 'USD', 2, -999],
+            ['000009.99', 'USD', 2, 999],
+            ['-000009.99', 'USD', 2, -999],
```

Comment:
Just to be sure that we don't wreak havoc with a `ltrim` in the future:

```suggestion
            ['-000009.99', 'USD', 2, -999],
            ['000', 'USD', 2, 0],
```